app/templates/components/forms/wizard/sessions-speakers-step.hbs

```diff
@@ -149,10 +149,10 @@
       </div>
       <div class="five wide field">
```

Comment:
@sumedh123 Remove the ui-dropdown it is not needed use the implementation from ticket-input instead
```
<div class="five wide field has popup" data-content="Uses the timezone of the event.">
  <label class="required" for="ticket_timezone">{{t 'Timezone'}}</label>
  {{input type='text' disabled=true value=timezone id='ticket_timezone' }}
</div>
```